src/Core/Test/Stub/Doctrine/FakeConnection.php

```diff
@@ -22,7 +22,7 @@ class FakeConnection extends Connection
      *
      * @throws Exception
      *
-     * @phpstan-ignore-next-line DBAL Connection uses psalm-consistent-constructor annotation,
+     * @phpstan-ignore-next-line parameter.missing DBAL Connection uses psalm-consistent-constructor annotation,
```

Comment:
```suggestion
     * @phpstan-ignore parameter.missing DBAL Connection uses psalm-consistent-constructor annotation,
```

that is the only one, we should use, as the `-next-line` and `-line` do not support those identifiers

Reply:
and I would adjust the comment, as we are not using psalm anymore :see_no_evil: